sentry-android-core/src/main/java/io/sentry/android/core/LoadClass.java

```diff
@@ -2,8 +2,16 @@
 
 import org.jetbrains.annotations.NotNull;
 
-final class LoadClass implements ILoadClass {
-  @Override
+/** An Adapter for making Class.forName testable */
+final class LoadClass {
```

Comment:
Following our other "util" classes, shouldn't it contain a static method?

Reply:
would need to use powermock to test it then, right?

Reply:
Mockito supports mocking static methods too: https://www.baeldung.com/mockito-mock-static-methods

But, perhaps its overengineering - having this method as non-static does not hurt.

Reply:
yeah, it'd be harder to mock the static methods, but it could, the class already exists so I will keep it like this